test/e2e/main_test.go

```diff
@@ -70,17 +70,23 @@ func TestClusterFormation(t *testing.T) {
 	}
 	// Register it
 	okd.CreateInfinispan(&spec, Namespace)
-	defer okd.DeleteInfinispan("cache-infinispan", Namespace)
+	defer okd.DeleteInfinispanAndWaitPodsCleanup("cache-infinispan", Namespace, "app=infinispan-pod", SinglePodTimeout)
```

Comment:
What about we leave the name of the method as is, `DeleteInfinispan`? Does it make sense to have a `DeleteInfinispan` method that does not wait for pod cleanup?

Reply:
yeah ok, I can do that

Reply:
done @galderz